### DISCUSSION TITLE
Samples

### DISCUSSION THREAD

**Discussion by igboyes** [virtool/.github#10] [Ideas]
Samples will be comprise one or more sample datasets.

# Example
- **Sample A**
  - Illumina Run 1
  - Sanger Run 1
  - PCR 1 (optional goal)

# Implications
- samples can be empty on creation; we will maintain the option of attaching a dataset to the sample when it is created
- this will allow workflows that use multiple datasets (eg. hybrid assembly)
- could include non-sequencing datasets (eg. PCR)
- we would have to rethink and reimplement sample creation workflow

**Comment by MikeERott** [comment]
For sample, include NA extraction run and type of NA
